system-addon/content-src/components/TopSites/TopSite.jsx

```diff
@@ -80,9 +91,10 @@ export class TopSiteLink extends React.PureComponent {
     let draggableProps = {};
     if (isDraggable) {
       draggableProps = {
-        draggable: true,
```

Comment:
this wasn't being used ?

Reply:
interesting, does it still work? https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/draggable